migrations/schemas/20221018135422-gitbook_click_collector.sql

```diff
@@ -0,0 +1,10 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS gitbook_click_collectors (
+    id SERIAL NOT NULL PRIMARY KEY,
+    command TEXT NOT NULL UNIQUE,
```

Comment:
i think we should split 2 column `command` and `action`
Ex: "https://mochibot.gitbook.io/mochi-bot/functions/nft-rarity-ranking-and-volume/nft-ticker"
-> command = nft-rarity-ranking-and-volume and action = nft-ticker